liquibase-core/src/main/java/liquibase/command/CommandFailedException.java

```diff
@@ -3,11 +3,13 @@
 public class CommandFailedException extends Exception {
     private final CommandResults results;
     private final int exitCode;
+    private final boolean hidden;
```

Comment:
Yeah, "hidden" isn't the best name since the error isn't hidden just how stacktrace gets logged.

Maybe:
- logStacktrace? (be explicit, but not 'future-proof' )
- unexpected? (more generic, default is true, only log stacktrace (or other future things) if unexpected). 
  - Would `expected` be nicer to avoid the double negative? Or `normal` etc.?
- ??

So far, I'd maybe lean to something more generic like `unexpected` and then have the javadoc something like: "CommandFailedException is thrown any time a command did not succeed. If it did not succeed due to normal and expected reasons, mark it as unexpected=false. If the CommandFailedException is marked as unexpected=true, the code calling the command may want to do additional logging or handling of the exception because it knows the command was surprised by the result."

Reply:
I'm going to go with `expected`, since the double negative of `unexpected` is a bit confusing.